### PR TITLE
[200707] Read antikythera instance's gear_templates dir

### PR DESCRIPTION
**Pull request by aDaichiOta**
https://acsmine.tok.access-company.com/redmine/issues/200707

### CODE REVIEW

File: lib/mix/gear.new.ex [hunk 1]
```diff
@@ -103,13 +103,22 @@ defmodule Mix.Tasks.Antikythera.Gear.New do
...
+             {:error, _} -> [dir1]
+             dir2        -> [dir1, dir2]
+           end
+    Enum.each(dirs, fn dir ->
+      template_dir = Path.join(dir, "gear_template")
+      Enum.each(template_files(template_dir), fn template_path ->
+        dest_rel_path = template_path |> Path.relative_to(template_dir) |> String.replace("gear_name", gear_name)
+        dest_path = Path.join(dest_dir, dest_rel_path)
+        content = EEx.eval_file(template_path, binding)
+        Mix.Generator.create_file(dest_path, content)
```
Comment (irisTa56) [inline lib/mix/gear.new.ex]: Could you tell me which is your plan?

1. instance repository has only template files which are not included in this repository.
2. instance repository has all the template files for a gear project.

For the latter case, if there is a conflict between the instance repository and this repository, [create_file/3](https://hexdocs.pm/mix/Mix.Generator.html#create_file/3) asks for user confirmation.
Is this as intended?

Comment (aDaichiOta) [reply lib/mix/gear.new.ex]: Thanks. It isn't my intention. I'll fix.

Comment (aDaichiOta) [reply lib/mix/gear.new.ex]: fixed: 2ba123db5cfe940aa335353b8423faf96d5d443d

**Comment by irisTa56** [comment]
LGTM!

**Comment by aMasakiTakahashi** [comment]
LGTM!

**Comment by ttokit** [comment]
LGTM!

**Pull request merged**